### DISCUSSION TITLE
Export page

### DISCUSSION THREAD

**Discussion by zakaria-chahboun** [wailsapp/wails#1079] [Q&A]
Hi! First of all thanks for this amazing project.
I'm a new to Wails and Go! 
I have a question, For example if i want to generare a pdf file from my front-end (turning an html page to pdf by a js library), How i can save it to a file by Go binding? 

Thanks 🙌

**Comment by leaanthony** [comment]
I would suggest sending the data to the backend via `EventEmit`. I do want to improve this moving of data, potentially by using the asset server as a cache so you could EG `runtime.CacheSave("myfile.pdf", pdfData);` and on the backend `runtime.OnCacheSave(func(filename string, data interface{}) {})` - something like that.
